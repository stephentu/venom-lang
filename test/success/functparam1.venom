def f{A}(a::A) -> A = 
  print (a);
  return a;
end
x = f{int}(32);
y::int = x;
print (y);
f{string}("foobar");
