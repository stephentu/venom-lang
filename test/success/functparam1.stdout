32
32
foobar
